src/libraries/System.Runtime.Serialization.Formatters/src/ILLink/ILLink.Substitutions.xml

```diff
@@ -0,0 +1,7 @@
+<linker>
```

Comment:
This new file needs to get referenced in the .csproj. The illink.targets doesn't support picking it up automatically. (since we didn't have a need initially)

So add 

```xml
<ItemGroup>
    <ILLinkSubstitutionsXmls Include="Path to file" />
</ItemGroup>
```